recipes-bsp/tegra-binaries/tegra-helper-scripts/make-sdcard.sh

```diff
@@ -451,6 +472,7 @@ if ! sgdisk "$output" --verify >/dev/null 2>&1; then
     echo "ERR: verification failed for $output" >&2
     exit 1
 fi
+create_filesystems || exit 1
```

Comment:
This should go after, rather than before, the `partprobe` below, and probably also belongs in the `if`/`then` section, since it only applies if writing to a block device.